package.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>rqt_gauges_2</name>
   <version>0.0.1</version>
   <description>Visualization plugin for several sensors.</description>
+  
+  <maintainer email="eloy.briceno@ekumenlabs.com">Eloy Bricneo</maintainer>
 
-  <maintainer email="eloy.briceno@ekumenlabs.com">Eloy Briceno</maintainer>
-
-  <license file="LICENSE">BSD Clause 3</license>
```

Comment:
Revert this change, It shouldn't be related.

PS: I tested locally just in case and it doesn't seem to affect.

Reply:
Done.